src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static Random random = new Random();
+
+    public static String getRandomColor() {
```

Comment:
static modifier is not required here, we call method from object
```suggestion
    public String getRandomColor() {
```